changelog/v0.28.0/supergraph-on-status.yaml

```diff
@@ -0,0 +1,4 @@
+changelog:
+  - type: NEW_FEATURE
```

Comment:
```suggestion
  - type: BREAKING_CHANGE
```

Reply:
i would read https://github.com/solo-io/go-utils/tree/master/changelogutils#changelog

Reply:
Sticking with NEW_FEATURE because the breaking change was rolled back if thats alright